src/main/java/org/cyclonedx/CycloneDxSchema.java

```diff
@@ -85,11 +90,19 @@ public JsonSchema getJsonSchema(CycloneDxSchema.Version schemaVersion, boolean s
     {
         final InputStream spdxInstream = getJsonSchemaAsStream(schemaVersion, strict);
 
+        final SchemaValidatorsConfig config = new SchemaValidatorsConfig();
+
+        final Map<String, String> offlineMappings = new HashMap<>();
+
+        offlineMappings.put("http://cyclonedx.org/schema/spdx.schema.json", getClass().getClassLoader().getResource("spdx.schema.json").toExternalForm());
+
+        config.setUriMappings(offlineMappings);
+
```

Comment:
All of this feels like it could be done in a constructor, etc..., but this class is abstract so I kept it this way for now. Long term it might be more efficient to move stuff into one time setup!